README.md

```diff
@@ -2,24 +2,250 @@
+
+* The GitHub mechanism serves as a consensus building medium for accepting changes
+once all dissenting opinions and questions have been resolved, providing an archive of
+the responses to those concerns and those individuals participating in the process.
+
+Templates are provided to assist in writing an Issue and/or Pull Request that includes an appropriate level of information for that type of request. Questions may be answered by anyone and participation from the broad PMIx community is encouraged and welcomed, as are suggestions for clarifications and corrections to the Standard. Feature requests are best presented in the form of a use-case as this assists both the community's understanding of the justification for providing the feature and in assessing the feasibility/magnitude of effort required to provide it.
+
+Proposed modifications to the Standard can come in several forms:
+
+* Addition of one or more new APIs and/or key-value attribute definitions
+
+* Changing the Standard classification of a current API and/or attribute
+
+* Deprecating an existing API and/or attribute
+
+* Clarifications and corrections to existing Standard language
+
+The Standardization Process remains essentially the same regardless of the type of modification being proposed, although the speed at which a modification progresses through the process may vary considerably based on the magnitude and controversial nature of the proposed change. Discussions regarding a proposal are conducted either on the Issue/PR itself or during the regular community teleconference. Teleconferences provide a high bandwidth discussion format and valuable synchronization mechanism to guide progress. Notes from the teleconferences are archived on the PMIx community's [wiki](https://github.com/pmix/pmix-standard/wiki#meeting-information).
+
+Note that a proposal can be withdrawn by the author(s) at any time for any reason up to the time the proposal is committed to the Standard, assuming acceptance.
+
+#### "Classes" of Standard
+The PMIx Standard is composed of a combination of APIs and key-value attributes. APIs are intentionally designed to be "lightweight" - i.e., the API signature is generic with a minimal number of fixed parameters in its signature. With few exceptions, APIs are required to include an array of key-value attributes in their signature to facilitate behavioral extensions without modifying the API signature. Thus, PMIx attempts to minimize deprecation and modification of APIs as use models evolve over time.
+
+Each API can designate a given attribute as either "**required**" or "**optional**" - this indicates whether or not a PMIx implementation of that API is required to support the attribute. PMIx implementations are requested to support optional attributes, but are not required to do so. This is intended to both provide flexibility to PMIx implementors (e.g., to tailor the implementation to their target audience) and to ease adoption of the Standard.
```

Comment:
These two paragraphs about required or optional APIs or attributes seem out of place. I think they belong in the standard document but not in this procedure document. What I mean is the ASC won't vote to decide if an attribute/API is required or optional, that would be part of the proposal

Reply:
I guess I can see it both ways - folks requested it for clarification of the rest of the language in the governance writeup. Note that we don't really have a "document" here. The governance description is just a readme at this stage.